src/sentry/api/endpoints/organization_member_details.py

```diff
@@ -97,6 +101,22 @@ def _is_only_owner(self, member):
 
         return True
 
+    def _serialize_member(self, member, request, allowed_roles=None):
+        context = serialize(
+            member,
+            serializer=OrganizationMemberWithTeamsSerializer()
+        )
+
+        if request.access.has_scope('member:admin'):
+            context['invite_link'] = member.get_invite_link()
+
+        if allowed_roles:
+            allowed_roles = [{'role': serialize(r, serializer=RoleSerializer()),
```

Comment:
Just pointing this out since it was just moved from where it was before, but the way this serializer is working is a tad bit expensive. The `serialize` function should take a list of all the roles and do them all in one pass, instead of invoking `serialize` on each one.

Reply:
Is this way ok? Because `serialize` wants a list of objects and not a list of dicts, I moved computing the `allowed` value inside of the serializer.

Reply:
Correct. I think how you changed it makes more sense. Thanks! :)